### ISSUE TITLE
thats not twrp for 13T Pro

### ISSUE THREAD

**Issue by ProcessDuck** [Scav-engeR/13T_Pro_Corot_Twrp#2]
its for redmi k60 ultra that has the same codename

**Comment by Scav-engeR** [comment]
Yes they use the same base source code as well as kernel source. So when talking about custom recoveries, they use the same custom recovery with minor config changes.  However because 13t pro is mtk its much harder to build twrp. still testing.
